--- configure	2018-07-16 02:29:43.000000000 -0400
+++ configure	2019-07-08 18:35:02.000000000 -0400
@@ -9792,8 +9792,6 @@
   pkg_failed=yes
 fi
     fi
-else
-	pkg_failed=untried
 fi
 if test -n "$PKG_CONFIG"; then
     if test -n "$NETTLE_LIBS"; then
@@ -9810,8 +9808,6 @@
   pkg_failed=yes
 fi
     fi
-else
-	pkg_failed=untried
 fi
 
 
@@ -9847,8 +9843,6 @@
 " "$LINENO" 5
 
 else
-	NETTLE_CFLAGS=$pkg_cv_NETTLE_CFLAGS
-	NETTLE_LIBS=$pkg_cv_NETTLE_LIBS
         { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
 $as_echo "yes" >&6; }
 	cryptolib="nettle"
@@ -9873,8 +9867,6 @@
   pkg_failed=yes
 fi
     fi
-else
-	pkg_failed=untried
 fi
 if test -n "$PKG_CONFIG"; then
     if test -n "$HOGWEED_LIBS"; then
@@ -9891,8 +9883,6 @@
   pkg_failed=yes
 fi
     fi
-else
-	pkg_failed=untried
 fi
 
 
@@ -9928,8 +9918,6 @@
 " "$LINENO" 5
 
 else
-	HOGWEED_CFLAGS=$pkg_cv_HOGWEED_CFLAGS
-	HOGWEED_LIBS=$pkg_cv_HOGWEED_LIBS
         { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
 $as_echo "yes" >&6; }
 	:
